tck/src/main/java/org/reactivestreams/tck/PublisherVerification.java

```diff
@@ -1108,6 +1112,84 @@ public void activePublisherTest(long elements, boolean completionSignalRequired,
   }
 
   /**
+   * Runs the test body with multiple {@code Publisher} lengths, collects the individual failures
+   * for those lengths and throws an {@code AssertionError} if there were any failures.
+   * If the body throws a {@code SkipException} with {@code isSkip()} {@code true}, the particular length test is ignored.
```

Comment:
link to `public void notVerified(String message) {`, perhaps as `for example as thrown by notVerified()` etc